tests/postgres_tests/test_ranges.py

```diff
@@ -283,6 +283,46 @@ def test_date_range(self):
             [objs[0]],
         )
 
+    def test_date_range_lower_inc(self):
+        objs = [
+            RangesModel.objects.create(dates=DateRange('2015-01-01', '2015-05-04')),
+            RangesModel.objects.create(dates=DateRange('2015-01-01', '2015-05-04', '()')),
+        ]
+        self.assertSequenceEqual(
+            RangesModel.objects.filter(dates__lower_inc=True),
+            [objs[0]],
+        )
+
+    def test_date_range_lower_inf(self):
+        objs = [
+            RangesModel.objects.create(dates=DateRange('2015-01-01', '2015-05-04')),
+            RangesModel.objects.create(dates=DateRange(None, '2015-05-04')),
+        ]
+        self.assertSequenceEqual(
+            RangesModel.objects.filter(dates__lower_inf=True),
+            [objs[1]],
+        )
+
+    def test_date_range_upper_inc(self):
+        objs = [
+            RangesModel.objects.create(dates=DateRange('2015-01-01', '2015-05-04')),
+            RangesModel.objects.create(dates=DateRange('2015-01-01', '2015-05-04', '()')),
+        ]
+        self.assertSequenceEqual(
+            RangesModel.objects.filter(dates__upper_inc=True),
+            [objs[0]],
+        )
+
+    def test_date_range_upper_inf(self):
+        objs = [
+            RangesModel.objects.create(dates=DateRange('2015-01-01', '2015-05-04')),
+            RangesModel.objects.create(dates=DateRange('2015-05-04', None)),
+        ]
+        self.assertSequenceEqual(
+            RangesModel.objects.filter(dates__upper_inf=True),
+            [objs[1]],
+        )
```

Comment:
Please move these tests to the `TestQuerying`. Try to reuse existing objects (see `TestQuerying.setUpTestData()`, it should be enough to add one new object with an infinite upper bound, e.g. `RangesModel(ints=NumericRange(1, None))`,

Reply:
ok